### DISCUSSION TITLE
New package candidate: is

### DISCUSSION THREAD

**Discussion by smebberson** [idearium/idearium-lib#140] [Ideas]
There is a bunch of code in the legacy branch that I keep needing to use. One such example is `is` at https://github.com/idearium/idearium-lib/blob/legacy/lib/util/is.js.

I think we should turn this into `@idearium/is`. Any thoughts?

**Comment by allanchau** [comment]
@smebberson I think @idearium/util would be better? It could contain a bunch of generic functions like this

**Comment by allanchau** [comment]
@smebberson Actually I think 2 libraries are needed here, 1 that compares objects/arrays etc and the other to check the environments.
The environments package could be called `@idearium/env` and instead of
```js
const isDevelopment = str => str.toLowerCase() === 'development';
```
It would look at the TARGET_ENV environment variable and look something like
```js
const isDevelopment = () => process.env.TARGET_ENV.toLowerCase() === 'development';
```
